.env.example

```diff
@@ -0,0 +1 @@
+PORT=5000
```

Comment:
Might be an idea to add a description of env vars to the README - obviously only this one to start with, but there'll probably be more